contribute/README.md

```diff
@@ -58,22 +58,22 @@ Before you start coding, please follow these basic guidelines:
 
 Depending on the release management of each repository, your contribution will be either included in a next release, or deployed live automatically.
 
-Except for GitHub, you can chat with most Ocean Protocol core developers in our [Discord](https://discord.gg/TnXjkR5) if you have further development questions.
+Except for GitHub, you can chat with most Ocean Protocol core developers in the [Discord](https://discord.gg/TnXjkR5) if you have further development questions.
 
 ### Develop a dApp or integration on top of Ocean Protocol
 
-We LOVE builders of dApps on Ocean! Nothing makes us feel prouder than seeing you create awesome things with our open-source tools.
+We LOVE builders of dApps on Ocean! Nothing makes us feel prouder than seeing you create awesome things with these open-source tools.
 
-If you need ANY help, then we're here to talk with you on [Discord](https://discord.gg/TnXjkR5) to give you advice. We're also consistently improving our docs to help you. And... you're here :)
+If you need ANY help, then we're here to talk with you on [Discord](https://discord.gg/TnXjkR5) to give you advice. We're also consistently improving these docs to help you. And... you're here :)
 
-### Improve our docs
+### Improve the docs
 
-Our docs repo can always be improved. If you found a mistake or have an improvement to make, then follow the steps in our [contribution workflow](./#your-contribution-workflow) to commit your changes.
+The docs repo can always be improved. If you found a mistake or have an improvement to make, then follow the steps in the [contribution workflow](./#your-contribution-workflow) to commit your changes.
 
 ### Apply for a developer job
 
 Do you REALLY love building on Ocean Protocol? Consider joining us full-time! Our openings are listed at [https://github.com/oceanprotocol/jobs](https://github.com/oceanprotocol/jobs).
 
 
 
-Check our [Community Page](https://www.oceanprotocol.com/community) for our social media links where you can join the buzz around Ocean or chat with us directly 😊 Toodles!
+Check the [Community Page](https://www.oceanprotocol.com/community) for the social media links where you can join the buzz around Ocean or chat with us directly 😊 Toodles!
```

Comment:
link does not exist anymore. We need to create an anchor in the site to the media links. https://github.com/oceanprotocol/site/issues/1730